projects/Magnetosphere/Magnetosphere.cpp

```diff
@@ -109,20 +109,19 @@ namespace projects {
       RP::get("Magnetosphere.constBgBX", this->constBgB[0]);
       RP::get("Magnetosphere.constBgBY", this->constBgB[1]);
       RP::get("Magnetosphere.constBgBZ", this->constBgB[2]);
-      RP::get("Magnetosphere.noDipoleInSW", dummy);
-      this->noDipoleInSW = dummy == 1 ? true:false;
+      RP::get("Magnetosphere.dipoleInSW", dummy);
+      this->dipoleInSW = dummy == 1 ? true:false;
       RP::get("Magnetosphere.dipoleScalingFactor", this->dipoleScalingFactor);
 
       RP::get("Magnetosphere.dipoleMirrorLocationX", this->dipoleMirrorLocationX);
 
       RP::get("Magnetosphere.dipoleType", this->dipoleType);
 
-      /* Enforce no dipole in solar wind with dipole type 4 */
-      if ((this->dipoleType == 4) && (!this->noDipoleInSW)) {
+      /* Remind the user of what they might want to do, current wisdom is that type 4 needs the dipole in the inflow cells too. */
+      if ((this->dipoleType == 4) && (!this->dipoleInSW)) {
          if(myRank == MASTER_RANK) {
-            std::cerr<<"Note: Initializing Magnetosphere with dipole type 4, enforcing no dipole in solar wind!"<<std::endl;
+            std::cerr<<"Note: Initializing Magnetosphere with dipole type 4, you should probably have the dipole in the solar wind and therefore set dipoleInSW = 1 (default)."<<std::endl;
          }
-         this->noDipoleInSW = true;
```

Comment:
Well, do we want to force it on actually then?

Reply:
No forcing, but error msg if not used (as suggested)? Users do, of course, read the outputs...